pkg/clusterapi/aws/actuator.go

```diff
@@ -507,30 +500,73 @@ write_files:
   owner: 'root:root'
   permissions: '0640'
   content: |
-    openshift_group_type: %[1]s
+    openshift_group_type: {{ .NodeType }}
+{{- if .IsNode }}
 - path: /etc/origin/node/bootstrap.kubeconfig
   owner: 'root:root'
   permissions: '0640'
   encoding: b64
-  content: %[2]s
+  content: {{ .BootstrapKubeconfig }}
+{{- end }}
 runcmd:
 - [ ansible-playbook, /root/openshift_bootstrap/bootstrap.yml]
+{{ if .IsNode -}}
```

Comment:
You really want this to strip the whitespace from the left instead of the right. It doesn't impact the result here, but it would make a big difference if the next line starts with spaces or tabs.